### DISCUSSION TITLE
Authorship and rights of sitelen pona glyphs

### DISCUSSION THREAD

**Discussion by jnpoJuwan** [lipu-linku/sona#90] [General]
# Authorship and rights

## _ku suli_

| Glyph                    | Authorship            | Licenced as CC0? | Ref.         |
|--------------------------|-----------------------|------------------|--------------|
| kin                      | jan Same              | Yes              | [^same]      |
| namako (crosshair)       | jan Same              | Yes              | [^same]      |
| namako (pepper)          | jan Sonja             | Yes?             |              |
| oko                      | jan Same              | Yes              | [^same]      |
| epiku (pona-emitters)    |                       |                  |              |
| epiku (up-arrow)         | jan Elemenopi?        | Yes?             | [^elemenopi] |
| jasima                   |                       |                  |              |
| kijete­san­takalu          | jan Same              | Yes              | [^same2]     |
| kipisi                   | jan Same              | Yes              | [^same]      |
| kokosila                 |                       |                  |              |
| ku                       | _Multiple people_     |                  |              |
| lanpan (jo-based)        |                       |                  |              |
| lanpan (pana-based)      |                       |                  |              |
| lanpan (incorrect)       |                       |                  |              |
| leko                     | jan Same              | Yes              | [^same]      |
| meso                     |                       |                  |              |
| misikeke (pill)          | jan Elemenopi         |                  | [^elemenopi] |
| misikeke (mortar-pestle) | jan Sala (Salurua)    | Yes              | [^sala]      |
| monsuta                  | jan Same              | Yes              | [^same]      |
| n                        | jan Sa                | Yes              | [^sa]        |
| soko (thick stem)        |                       |                  |              |
| soko (thin stem)         |                       |                  |              |
| soko (annulus)           |                       |                  |              |
| tonsi                    | Xanthe (@satyrshorns) | Yes              | [^xanthe]    |

[^elemenopi]: jan Elemenopi [u/ElemenopiTheSequel] (16 August 2020). ["Here are the official glyphs for the 1b words in the NA"](https://reddit.com/r/OffThePu/comments/iahhtc/here_are_the_official_glyphs_for_the_1b_words_in). r/OffThePu. Reddit.
[^sa]: jan Sa [@green.guy] (28 January 2021). [[Message on ma pona pi toki pona]](https://discord.com/channels/301377942062366741/301377942062366741/804397312545783920). Discord. "n!"
[^sala]: jan Sala [salarua] (27 July 2021). [[Message on ma pona pi toki pona]](https://discord.com/channels/301377942062366741/375591429608570881/869476620006723604). Discord.
[^same]: jan Same. (16 October 2016). "[Sitelen pona glyphs for new and apocryphal words](http://forums.tokipona.org/viewtopic.php?t=2616)". Toki Pona Forums.
[^same2]: James Flear [jan Same] (12 July 2019). ["toki! After a week of voting, I present to you **the winners of the *“New sitelen pona glyph contest”.***"](https://www.facebook.com/groups/sitelen/posts/2367140973340274). Facebook. [Archived](https://web.archive.org/web/20230809220109/https://www.facebook.com/groups/sitelen/posts/2367140973340274) from the original on 9 August 2023. 
[^xanthe]: Mx Xanthe Tynehorne [satyrshorns] (30 March 2020). [[Message on ma pona pi toki pona]](https://discord.com/channels/301377942062366741/301378960468738050/694279558932725800). Discord.

---

# Appendix: Release of rights by person

## jan Elemenopi

<img src="https://github.com/lipu-linku/sona/assets/48970145/fa91a111-9e1e-4cf7-8602-cd6870c33fe6" height="200"/>  \
Screenshot taken by jan Kekan San on 2024-06-23

## jan Sala

<img src="https://github.com/lipu-linku/sona/assets/48970145/bd33e058-6877-47f2-bc29-8040bb411963" width="200"/>  \
Screenshot taken by jan Sala on 2024-06-21

## jan Same

<img src="https://github.com/user-attachments/assets/e10444be-4831-400d-9019-137ada1be3da" width="200"/>  \
Screenshot taken by jan Kekan San on 2024-07-26

[janSame_written_approval_for_sitelen_pona_glyphs.pdf](https://github.com/user-attachments/files/16393917/janSame_written_approval_for_sitelen_pona_glyphs.pdf)
Handwritten release of the copyrigth by jan Same

## Xanthe

<img src="https://github.com/lipu-linku/sona/assets/48970145/e96f5067-8101-4044-aa13-91e4d83e7118" width="200"/>  \
Screenshot taken by jan Kekan San on 2024-06-22

**Comment by yan-Mayeka** [comment]
Here is a photo of the originaol creator of the soko wide stem variant releasing it to cc0 1.0 universal
![image](https://github.com/user-attachments/assets/a95989b0-b505-40db-8e62-602fe190feea)
here is a link to the first instance of this glyph variant
https://discord.com/channels/301377942062366741/301378960468738050/803717131413618758

Here is a photo of the originaol creator of the soko thin stem variant releasing it to cc0 1.0 universal
![image](https://github.com/user-attachments/assets/519c70a9-28c2-42c9-9b44-37334ef901da)
here is a link to the first instance of this glyph variant
https://discord.com/channels/301377942062366741/375591429608570881/713760909545177170

Here is a photo of the originaol creator of the meso glyph releasing it to cc0 1.0 universal
![image](https://github.com/user-attachments/assets/f9832086-eb92-43a1-b054-1985501b6bcc)
Here is a link to the first instance of this glyph
https://discord.com/channels/301377942062366741/375591429608570881/793491675212546069

**Comment by gregdan3** [comment]
https://discord.com/channels/948448338662010910/1191946786936459325/1280724729342197853
![image](https://github.com/user-attachments/assets/6f0cf705-912d-4275-a843-6a8446c18a24)

https://discord.com/channels/301377942062366741/340307145373253642/934077537313779793
![image](https://github.com/user-attachments/assets/ffbe6996-65f6-4e58-b8ff-b9fe1f58a9a1)


https://discord.com/channels/301377942062366741/340307145373253642/934078987364360264
![image](https://github.com/user-attachments/assets/d3f3f6e7-c0c0-454a-af0a-97824f511657)

**Comment by yan-Mayeka** [comment]
Here is a photo of the originaol creator of the lanpan jo based version releasing it to cc0 1.0 universal
![Screenshot (897)](https://github.com/user-attachments/assets/d52e965c-cc98-4385-81ec-388d77af2245)

Here is a link to someone forwarding what the creator came up with while in a conversation discussing what the glyph for lanpan should be
https://discord.com/channels/301377942062366741/301377942062366741/814159612873015326

**Comment by gregdan3** [comment]
https://discord.com/channels/948448338662010910/951966419283742750/1283523255981838348

![image](https://github.com/user-attachments/assets/f1fe8905-f93e-4798-a8ce-aa06b2d46e2f)


https://discord.com/channels/948448338662010910/951966419283742750/1030300874305703977

![image](https://github.com/user-attachments/assets/ac64b6fe-bbb6-4ce1-9b4c-5b1190b11d9b)


:)

**Comment by yan-Mayeka** [comment]
Here is a photo of the originaol creator of the annulus soko releasing it to cc0 1.0 universal
![image](https://github.com/user-attachments/assets/4dbe8d44-0910-492c-95b4-d65acd736b1e)
first public usage: https://discord.com/channels/301377942062366741/340307145373253642/933245343506780211

**Comment by anna328p** [comment]
As the creator of the ku glyph, I hereby dedicate it to the public domain by marking the work with [CC0 1.0 Universal](https://creativecommons.org/publicdomain/zero/1.0/).

![Screenshot_20250518-111727.png](https://github.com/user-attachments/assets/fb1aeb55-a22e-4824-9815-6d5f2328d970)
